content/automate/ManageIQ/System/Event/EmsEvent/ORACLE.class/__class__.yaml

```diff
@@ -0,0 +1,433 @@
+---
```

Comment:
I have no idea what this file does :laughing: I copied it from another provider class